source/chapters/introduction.rst

```diff
@@ -40,25 +40,24 @@ New in Mixxx |version|
    RJ Ryan <rryan@mixxx.org>
    S.Brandt <s.brandt@mixxx.org>
 
-* 4 decks (including 4 vinyl control inputs if you want to use 4 turntables)
+* 4 decks
 * Master sync
-* Key detection and pitch shifting (independent of tempo) for harmonic mixing.
-* 4 effect units for built-in effects (this also introduces all the necessary
-  plumbing for LV2/VST, effect send/receive, and advanced chaining in a future
-  release)
-* Beatloop rolls (now you can create beatloops and automatically jump to where
```

Comment:
Is this different from the `beat jumping and loop move` we have listed below? If so please readd the line.